### PR TITLE
Add button and popover to show legend for creator 2D

### PR DESCRIPTION
**Pull request by whyang0808**
AddPopoverButton
Hi, I've added the popover, kindly review.  😄

### CODE REVIEW

**Comment by whyang0808** [comment]
#14

File: src/styles/style.scss [hunk 1]
```diff
@@ -365,6 +365,34 @@ main {
   pointer-events: none;
   color: white;
 }
+.path.dark-gray-legend {
```
Comment (Nienormalny) [inline src/styles/style.scss]: Will be nice to implement all popover style inside `styles/molecules/popover.scss` (thus file doesn't exist yet) and you can put there standard style for popovers, together with extra styling for legend. :+1:

File: src/styles/style.scss [hunk 2]
```diff
@@ -365,6 +365,34 @@ main {
...
+  background-color: $wall-color;
+}
+.path.red-legend {
```
Comment (Nienormalny) [inline src/styles/style.scss]: you can put all extra classes for `.path` inside `.path`.
```
.path {
&.red-legend {...}
...
}
```

File: src/styles/style.scss [hunk 3]
```diff
@@ -434,6 +462,44 @@ label {
   }
 }
 
+#buttons-container2D {
+  display: flex;
+  justify-content: space-between;
+}
+
+.popover-wrapper {
```
Comment (Nienormalny) [inline src/styles/style.scss]: Will be perfect to put it inside `popover.scss` file as standard style for popovers.

File: src/styles/style.scss [hunk 4]
```diff
@@ -434,6 +462,44 @@ label {
...
+  position: relative;
+}
+.popover-content {
```
Comment (Nienormalny) [inline src/styles/style.scss]: Same here.

File: src/styles/style.scss [hunk 5]
```diff
@@ -434,6 +462,44 @@ label {
...
+  position: absolute;
+  left: -180px;
+  transform: translate(0, 40px);
+  border-radius: 10px;
+  background-color: $gray;
+  padding: 1.5rem;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
+  width: 185px;
+}
+.popover-content:before {
```
Comment (Nienormalny) [inline src/styles/style.scss]: Same here.

File: src/styles/style.scss [hunk 6]
```diff
@@ -434,6 +462,44 @@ label {
...
+  position: absolute;
+  z-index: -1;
+  content: "";
+  right: 18px;
+  top: -8px;
+  border-style: solid;
+  border-width: 0 10px 10px 10px;
+  border-color: transparent transparent $gray transparent;
+}
+.popover-wrapper:hover .popover-content {
```
Comment (Nienormalny) [inline src/styles/style.scss]: Same here.

File: src/styles/style.scss [hunk 7]
```diff
@@ -434,6 +462,44 @@ label {
   }
 }
 
+#buttons-container2D {
```
Comment (Nienormalny) [inline src/styles/style.scss]: You can put style for button inside `buttons.scss`.

File: src/styles/style.scss (hunk 3 above)
Comment (Nienormalny) [reply src/styles/style.scss]: Please check comment for line: 368, to get correct path and file structure for styles.

**Pull request merged**